src/main/java/seedu/realtime/logic/Logic.java

```diff
@@ -28,11 +28,11 @@ public interface Logic {
     CommandResult execute(String commandText) throws CommandException, ParseException;
 
     /**
-     * Returns the AddressBook.
+     * Returns the realTime.
```

Comment:
Should change to this
```suggestion
     * Returns the RealTime.
```